internal/tiltfile/tiltfile_state.go

```diff
@@ -108,6 +108,8 @@ const (
 	kustomizeN    = "kustomize"
 	helmN         = "helm"
 	listdirN      = "listdir"
+	jsonDecodeN   = "json_decode"
```

Comment:
Is it worth being consistent about putting "json" at either the start or end of these names, or is there a tradeoff here?

Reply:
The consistency in my head was if the function is `read_file` then the thing that reads json from a file should be `read_json` or at least `read_json_file`.

And I'm now realizing I stole `json_decode` [from PHP](http://php.net/manual/en/function.json-decode.php) 😬